db/imgcodec/tiff/tiled/make.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -xe
+
+CAT="cat-111793_640"
+ORIG_CAT="$DALI_EXTRA_PATH/db/single/tiff/0/$CAT.tiff"
+
+convert $ORIG_CAT -define tiff:tile-geometry=16x48 -compress lzw ${CAT}_tiled_16x48.tiff
+
+convert $ORIG_CAT -define tiff:tile-geometry=1024x1024 -compress lzw ${CAT}_tiled_1024x1024.tiff
+
+convert $ORIG_CAT -define tiff:tile-geometry=32x32 -compress lzw \
```

Comment:
I didn't manage to open/read this image with any common tool

Reply:
There's something wrong with this file, I'll investigate. Thanks!

Reply:
I decided to drop support for tiled TIFFs with exotic bit depths, at least for now. I couldn't find any software that would create/view those correctly.